__tests__/app.test.js

```diff
@@ -385,4 +386,16 @@ describe("app", () => {
       });
     });
   });
+
+  describe("GET - /api/articles/:article_id (comment_count)", () => {
+    test("GET: 200 - should return the total amount of all comments for a specific article_id", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const article = body.article;
+          expect(typeof article.comment_count).toBe("number");
```

Comment:
You could test for the actual number of comments here so you're completely sure your query is working